neuraxle/steps/flow.py

```diff
@@ -0,0 +1,199 @@
+            'Fit method is not supported for {0}, because it inherits from ForceHandleMixin. Please use handle_fit instead.'.format(
+                self.name))
+
+    def fit_transform(self, data_inputs, expected_outputs=None) -> 'ForceHandleMixin':
+        raise Exception(
+            'Fit transform method is not supported for {0}, because it inherits from ForceHandleMixin. Please use handle_fit_transform instead.'.format(
+                self.name))
+
+
+class Nullify(ForceHandleMixin, MetaStepMixin, BaseStep):
+    """
+    A wrapper to nullify a step : nullify its hyperparams, and also nullify all of his behavior.
+
+    Example usage :
+
+    .. code-block:: python
+
+        p = Pipeline([
+            Nullify(Identity())
+        ])
+
+    """
+
+    def __init__(self, wrapped: BaseStep):
+        ForceHandleMixin.__init__(self)
+        MetaStepMixin.__init__(self, wrapped)
+        BaseStep.__init__(self)
+
+    def handle_fit(self, data_container: DataContainer, context: ExecutionContext) -> ('BaseStep', DataContainer):
+        """
+        Nullify wrapped step hyperparams, and don't fit the wrapped step.
+
+        :param data_container: data container
+        :type data_container: DataContainer
+        :param context: execution context
+        :type context: ExecutionContext
+        :return: step, data_container
+        :type: (BaseStep, DataContainer)
+        """
+        self._nullify_hyperparams()
+        return self, data_container
+
+    def handle_fit_transform(self, data_container: DataContainer, context: ExecutionContext) -> (
+            'BaseStep', DataContainer):
+        """
+        Nullify wrapped step hyperparams, and don't fit_transform the wrapped step.
+
+        :param data_container: data container
+        :type data_container: DataContainer
+        :param context: execution context
+        :type context: ExecutionContext
+        :return: step, data_container
+        :type: (BaseStep, DataContainer)
+        """
+        self._nullify_hyperparams()
+        return self, data_container
+
+    def handle_transform(self, data_container: DataContainer, context: ExecutionContext) -> DataContainer:
+        """
+        Nullify wrapped step hyperparams, and don't transform the wrapped step.
+
+        :param data_container: data container
+        :type data_container: DataContainer
+        :param context: execution context
+        :type context: ExecutionContext
+        :return: step, data_container
+        :type: DataContainer
+        """
+        self._nullify_hyperparams()
+        return data_container
+
+    def _nullify_hyperparams(self):
+        """
+        Nullify wrapped step hyperparams using hyperparams_space.nullify().
+        """
+        hyperparams_space = self.wrapped.get_hyperparams_space()
+        self.wrapped.set_hyperparams(hyperparams_space.nullify())
+
+
+CHOOSE_ONE_OR_MANY_STEPS_OF_CHOICE_HYPERPARAM = 'choice'
+
+
+class ChooseOneOrManyStepsOf(Pipeline):
```

Comment:
Oops, in my head this was inheriting from FeatureUnion to do things in parallel, not in series.

Reply:
I think that fixing that makes the class not change very much tho.

Reply:
Pipeline has the implementation with handle methods... what are you suggesting exactly ? you want me to inherit from FeatureUnion ?

Reply:
Yes, inherit from FeatureUnion instead of Pipeline. Feature Union will need the handle methods.